utils/lineinfile/lineinfile.go

```diff
@@ -0,0 +1,253 @@
+package lineinfile
```

Comment:
Hm, did you consider using something existing? There is for instance this library: https://github.com/sergeymakinen/go-systemdconf. Not sure if works for the daemon config files, the systemd.syntax man page says it has the same syntax so might work actually.

Reply:
Yeah, I did some research there, most of them seemed too bloated, complex and/or unmaintained, so I didn't want to get trapped in the future by introducing such dependency (e.g. this one depends on 5 years old release of a **CSS** processing library). In the end I went with the "lineinfile" concept, which is powerful enough it's able to drive configuration changes by many Ansible roles in the real world, and it's easy to understand and troubleshoot.